client/sites-dashboard-v2/style.scss

```diff
@@ -204,7 +208,13 @@
 
 		.dataviews-view-table-wrapper {
 			overflow-y: auto;
-			max-height: calc(100vh - 326px); /* 326px is the size of all the height above and below the dataview, from the edges of the screen, plus 10px offset from the padding. Currently below 9px + 309 above */
+			// Without masterbar
+			max-height: calc(100vh - 285px); /* 285px is the size of all the height above and below the dataviews-view-table-wrapper */
+
+			div.is-group-sites-dashboard:not(.has-no-masterbar) & {
+				// With masterbar
+				max-height: calc(100vh - 347px); /* 347px is the size of all the height above and below the dataviews-view-table-wrapper */
```

Comment:
Probably, this could go at `dotcom-style.scss` file

Reply:
hmm I don't quite understand the rational for where the CSS is placed between the different files. 

But I've taken all of the code responsible for setting `overflow-y` and `max-height` on `.dataviews-view-table-wrapper` and `.is-group-sites-dashboard` and moved it to a section in `dotcom-style.scss` so that it's all in one place. There is also related code in:
`client/a8c-for-agencies/sections/sites/sites-dashboard/style.scss` but I've left that as is.

Reply:
actually NVM I think I understand what you were asking for better after reading your other comment, I've only moved the masterbar specific code to dotcom-style.scss